src/app/nation/nation.component.html

```diff
@@ -36,6 +36,7 @@
   <ng-container matColumnDef="action">
     <mat-header-cell *matHeaderCellDef i18n> Action </mat-header-cell>
     <mat-cell *matCellDef="let element">
+      <button type="button" mat-raised-button color="primary" (click)="viewResources(element.name)" i18n>View Resources</button>
```

Comment:
do we need type="button"

Reply:
exactly we don't need type="button" as it's tag <button> already says it is of type button.